us-west1/config.tf

```diff
@@ -4,7 +4,7 @@ terraform {
     organization = "el-brando"
 
     workspaces {
-      name = "gcp-us-west1"
+      name = "gcp-us-west1-josse-test"
```

Comment:
you do not need to touch this file.  the workspace is the Terraform cloud workspace.  that will always be gcp-us-west1